lib/fetch/rss/fetchRssProxy.ts

```diff
@@ -0,0 +1,23 @@
+import { IRss } from '@interfaces/data';
+import RssProxyError from './RssProxyError';
+
+const fetchRssProxy = async (feedUrl: string) => {
+  const apiProxyBaseUrl =
+    process.env.API_BASE_URL || 'https://play.prx.org/api';
```

Comment:
Ah, cool, there is a default.  I was getting 400s before I set the `API_BASE_URL` env, but turns out it was just prod 429-ing again.